### PR TITLE
CAMEL-15293 Cassandraql: Upgrade datastax driver to 4.7.2 (from 3.7.2)

### PR DESCRIPTION
**Pull request by JiriOndrusek**
Issue: https://issues.apache.org/jira/browse/CAMEL-15293

There are a lot of exceptions org.hyperic.sigar.SigarException during integration tests (https://github.com/jsevellec/cassandra-unit/issues/225)

!There has to be also a fix for camel-karaf (hence this PR is only a draft so far)

[ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/CAMEL) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
[ ] Each commit in the pull request should have a meaningful subject line and body.
[ ] If you're unsure, you can format the pull request title like `[CAMEL-XXX] Fixes bug in camel-file component`, where you replace `CAMEL-XXX` with the appropriate JIRA issue.
[ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
[ ] Run `mvn clean install -Psourcecheck` in your module with source check enabled to make sure basic checks pass and there are no checkstyle violations. A more thorough check will be performed on your pull request automatically.
Below are the contribution guidelines:
https://github.com/apache/camel/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: components/camel-cassandraql/pom.xml [hunk 1]
```diff
@@ -45,8 +45,13 @@
...
         <dependency>
-            <groupId>com.datastax.cassandra</groupId>
-            <artifactId>cassandra-driver-core</artifactId>
+            <groupId>com.datastax.oss</groupId>
+            <artifactId>java-driver-core</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.datastax.oss</groupId>
+            <artifactId>java-driver-query-builder</artifactId>
+            <version>4.7.2</version>
```
Comment (oscerd) [inline components/camel-cassandraql/pom.xml:54]: This should go in parent pom

Comment (JiriOndrusek) [reply components/camel-cassandraql/pom.xml:54]: sounds great, I'll rework it and I think that tests could be allowed on CI with testcontainers. (I'v kept cassandra-util as it was used before)

Comment (JiriOndrusek) [reply components/camel-cassandraql/pom.xml:54]: I'll fix that version, I've missed it

**Comment by omarsmak** [comment]
Hey @JiriOndrusek, is this still WIP PR? :)

**Comment by JiriOndrusek** [comment]
Hi @omarsmak  , yes. I've just started to refactor tests to use testcontainers. 
But on the other hand this PR could be merged and testcontainers could be added as a separate issue.
@oscerd what do you think?

**Comment by oscerd** [comment]
Sounds good. But please check if it works in camel-karaf

**Comment by JiriOndrusek** [comment]
Karaf needs some changes. I'll start with karaf to finish this PR

**Comment by JiriOndrusek** [comment]
Karaf PR is prepared: https://github.com/apache/camel-karaf/pull/12

**Comment by JiriOndrusek** [comment]
@oscerd Karaf fix is prepared.
I've created an issue for refactor to use testcontainers - https://issues.apache.org/jira/browse/CAMEL-15331
Is it possible to withdraw change request here?

**Pull request merged**